src/coordinator/storage/block.go

```diff
@@ -69,6 +70,19 @@ func (m multiSeriesBlock) Meta() block.Metadata {
 	return m.meta
 }
 
+func (m multiSeriesBlock) Steps() int {
+	// If series has lesser points then it should return NaNs
+	return m.meta.Bounds.Steps()
+}
+
+func (m multiSeriesBlock) Series() int {
+	if len(m.seriesList) == 0 {
```

Comment:
nit:  unnecessary

Reply:
had changed it and forgot to remove this condition